src/pages/MainPage/ContactsSection.jsx

```diff
@@ -8,75 +7,93 @@ import YoutubeIcon from '../../assets/img/youtubeIcon.svg';
 
 const ContactsSection = () => {
   return (
-    <section className="contacts">
-      <div className="contacts__wrapper">
-        <h2 className="contacts__title">CONTACT US </h2>
-        <p className="contacts__text">
+    <section className="min-h-[41.81rem] pt-48">
+      <div className="font-fourth container mx-auto">
```

Comment:
`font-fourth` is used, but it isn't defined in this branch, you need to pull master branch

Reply:
I thought that I will pull all branches step by step, starting with AboutSection. So when I added 'font-fourth' in previous section, I thought that it'll be right to w8 for merging previous sections with master. And when I will merge this branch it will be able to see the styles in cfg.

Is it a good method for such situations?

Reply:
Yes, you are right, you need to merge PR containing necessary values in config to master and then pull master to this branch